script/check_elasticsearch.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+# This script try to connect to elastic search and access the DB
+# return 0 if success / return 1 if error
+
+# exit if no parameter given
+if [ "$1" == "" ]
+then
+    echo "You should pass local.ini as argument"
+    exit 1
+fi
+
+# parse elasticsearch_host from local.ini
+elasticsearch_host="$(cat $1 | grep "elasticsearch_host")"
+elasticsearch_host=${elasticsearch_host// /}
+elasticsearch_host=${elasticsearch_host//elasticsearch_host=/}
+
+# parse elasticsearch_port from local.ini
+elasticsearch_port="$(cat $1 | grep "elasticsearch_port")"
+elasticsearch_port=${elasticsearch_port// /}
+elasticsearch_port=${elasticsearch_port//elasticsearch_port=/}
+
+# parse elasticsearch_index from local.ini
+elasticsearch_index="$(cat $1 | grep "elasticsearch_index")"
+elasticsearch_index=${elasticsearch_index// /}
+elasticsearch_index=${elasticsearch_index//elasticsearch_index=/}
+
+# parse elasticsearch_version from local.ini
+elasticsearch_version="$(cat $1 | grep "elasticsearch_version")"
+elasticsearch_version=${elasticsearch_version// /}
+elasticsearch_version=${elasticsearch_version//elasticsearch_version=/}
+
+ELASTICSEARCH_URL=http://${elasticsearch_host}:${elasticsearch_port}
+
+CURL="$(curl --fail --silent --show-error ${ELASTICSEARCH_URL} 2>&1)"
+
+# if curl didn't succeed 
+if [ $? != 0 ]
+then
+    echo "Fail to connect to elatic search"
+    echo "${CURL}"
+    exit 1
+fi
+
+# clear line to keep cluster_name value only
+CLUSTER_NAME=$(echo "${CURL}" | grep "cluster_name")
```

Comment:
This is a good candidate for jq. We'll make sure that the servers have it installed.
Then:
```suggestion
CLUSTER_NAME=$($CURL | jq '.cluster_name' | tr -d "\"" )
```